inst/silverstein/R/Carrots.R

```diff
@@ -0,0 +1,33 @@
+# R6 Class Definitions for testing
+
+#' @title Carrots 
+#' @name Carrots 
+#' @family Carrots 
+#' @description Class in which every method has an equal number of package function references 
+#' @importFrom R6 R6Class
+#' @export
+Carrots  <- R6Class(
+  classname = "Carrots ",
+  
+  # necessary to guarantee all class methods have precisely one reference
+  cloneable = F, 
```

Comment:
Minor style nit: we prefer longhand `TRUE`/`FALSE` for readability and making sure it doesn't collide with variables.

Reply:
I understand completely.  I often find myself switching back and forth between the two.